src/lib/dev-environment/dev-environment-lando.js

```diff
@@ -352,3 +352,16 @@ export async function validateDockerInstalled() {
 		throw Error( 'docker-compose could not be located! Please follow the following instructions to install it - https://docs.docker.com/compose/install/' );
 	}
 }
+
+export async function validateDockerUser() {
+	const lando = new Lando( getLandoConfig() );
+	await lando.bootstrap();
+
+	const docker = lando.engine.docker;
+	lando.log.verbose( 'Fetching docker info to verify user is in docker group' );
+	try {
+		await docker.info();
+	} catch ( error ) {
+		throw Error( 'Failed to connect to docker. Please follow the following instructions to add yourself to the docker group - https://docs.docker.com/engine/install/linux-postinstall/' );
```

Comment:
What do you think about this fix?

```suggestion
        if ( os.platform() === 'linux' ) {
    		throw Error( 'Failed to connect to docker. Please follow the following instructions to add yourself to the docker group - https://docs.docker.com/engine/install/linux-postinstall/' );
    	} else {
    	    throw Error( 'Please contact the support' ); // Obviously, we need a different text here
    	}
```